.travis.yml

```diff
@@ -19,7 +19,7 @@ sudo: false
 matrix:
   fast_finish: true
   include:
-   - node_js: "6"
+   - node_js: "node"
```

Comment:
why this change?

Reply:
There's no benefit to pinning the linter on node 6.

Reply:
is there a benefit to pinning the karma tests, seeing as they run in a browser?

Reply:
Only if the karma tools are c deps. I don't know much about karma.

Reply:
i dont think it makesany difference, but no point in changing it now